### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by Alhajideen**
Hi,
## Bookstore
In this milestone, I was able to achieve the following:
- Instantiate react app using CRA
- Remove react boilerplate
- Create the basic design with minute styles.
- Use Routes and links to navigate
- Use professional git-flow
- Add descriptive Readme

Please kindly review 🧑‍💻

### CODE REVIEW

File: src/Components/BookContainer.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  };
+  return (
+    <div>
+      <section className="bookContainer">
+        <div className="book-details">
+          <h3>Empty</h3>
+          <Book props={bookArr} />
+        </div>
+        <div className="book-actions">
+          <button type="button">Remove</button>
```
Comment (codepantha) [inline src/Components/BookContainer.js]: - [ ] Please render a list of books using the `Book` component which you have created. In order to achieve this, `bookArr` should be an array of book objects. Please iterate through this array and pass in each book as a prop to the `Book` component in order to render them correctly👍

File: README.md [hunk 2]
```diff
@@ -1,70 +1,63 @@
...
+## Acknowledgments
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+- Hat tip to [Stanley SIMEON](https://github.com/sstanleySimeon/) for design inspiration
 
-### Advanced Configuration
+## 📝 License
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+This project is [MIT](./LICENSE) licensed.
```
Comment (codepantha) [inline README.md]: - [ ] I noticed that the link to the MIT license is broken. Please fix it by adding a LICENSE to your project 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @Alhajideen 👋, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/BookContainer.js (hunk 1 above)
Comment (Alhajideen) [reply src/Components/BookContainer.js]: Thank you. Done

File: README.md (hunk 2 above)
Comment (Alhajideen) [reply README.md]: Thank you. License added

**Review by OLIPLICHE — APPROVED** [review]
Hi @Alhajideen, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# Highlights

- Implemented the changes from previous review👏 👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**